.github/workflows/ci_workflows.yml

```diff
@@ -18,56 +22,35 @@ jobs:
             python: 3.x
             toxenv: codestyle
 
-          - name: Python 3.8 with minimal dependencies
+          - name: Python 3.10 with astropy data and coverage
             os: ubuntu-latest
-            python: 3.8
-            toxenv: py38-test
+            python: '3.10'
+            toxenv: py310-test-cov
+            toxargs: -v
+            toxposargs: --open-files --remote-data=astropy
 
-          - name: Python 3.9 with all optional dependencies
```

Comment:
I removed the redundant jobs. `alldeps` is a lie, so some of the jobs are actually equivalent. Quantity =/= quality